compiler-rt/lib/hwasan/hwasan_report.cpp

```diff
@@ -323,6 +323,22 @@ static uptr GetGlobalSizeFromDescriptor(uptr ptr) {
 
 void ReportStats() {}
 
+constexpr uptr kShadowDumpWidth = 16;
```

Comment:
This should be used in `PrintTagInfoAroundAddr`. I would call it `kDumpWidth`, because it gets used for shadow and short dumps.

Reply:
Done